platform/windows/crash_handler_windows.cpp

```diff
@@ -34,6 +34,7 @@
 #include "core/os/os.h"
 #include "core/string/print_string.h"
 #include "core/version.h"
+#include "launch_debugger_windows.h"
```

Comment:
We're soon going to merge https://github.com/godotengine/godot/pull/75932 so you can move it up to match that style guide.